package.json

```diff
@@ -19,13 +18,26 @@
     "react-scripts": "3.4.3",
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "serverless-finch": "^2.6.0",
+    "serverless-single-page-app-plugin": "file:serverless-single-page-app-plugin"
```

Comment:
These packages are only needed locally, so they can be moved to devDependencies.